### PR TITLE
ba-134 Consolidate field names between inventory and recipe

### PR DESCRIPTION
**Pull request by lyio**
This will break with existing inventory items
- Rename count in inventory items to amount
- Rename measurement in inventory items to unit

### CODE REVIEW

File: app/test/socobo-inventory/socobo-inventory-behavior.html [hunk 1]
```diff
@@ -32,15 +32,15 @@
       });
 
       test('valid items are validated correctly', function(done) {
-        var validItem = item.extend({name: "Valid name", count: "4", measurement: {__firebaseKey__: "0", value: "pcs"}});
+        var validItem = Object.assign(item, {name: "Valid name", amount: "4", unit: {__firebaseKey__: "0", value: "pcs"}});
```
Comment (flashback2k14) [inline app/test/socobo-inventory/socobo-inventory-behavior.html]: Object.assign is ES6!?

Comment (lyio) [inline app/test/socobo-inventory/socobo-inventory-behavior.html]: Yes, but it works in the current FF and Chrome and that is good enough for running the tests, or not?

Comment (flashback2k14) [inline app/test/socobo-inventory/socobo-inventory-behavior.html]: Yes, but you have a Workung solution that doesn't break oder Browser.

**Comment by flashback2k14** [comment]
@lyio following error apears if i click on the edit button

bug 1

**Comment by lyio** [comment]
That is due to the fact that the old items in the database have not been migrated.

**Comment by flashback2k14** [comment]
show dialog doesn't show amount and measurement

bug 2

**Comment by flashback2k14** [comment]
lgtm

**Comment by lyio** [comment]
lgtm

**Comment by lyio** [comment]
lgtm

**Pull request merged**